bin/reports/compile_uncommitted_holdings.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "reports/uncommitted_holdings"
+require "optparse"
+require "services"
+
+Services.mongo!
+
+def main(args)
+  criteria = parse_opts(args)
+  report = Reports::UncommittedHoldings.new(criteria)
+  puts report.header.join("\t")
+  report.run do |record|
+    puts record.to_s
+  end
+end
+
+def parse_opts(args)
```

Comment:
https://github.com/hathitrust/holdings-backend/tree/HT-3273-consolidate-bin-scripts-with-cli
can take care of most of this so don't sweat the details in here.